pkg/config/config.go

```diff
@@ -44,8 +47,9 @@ type Config struct {
 	DisableKeyvault                                   bool
 	MSIClientID, TenantID                             string
 	Cloud, Location                                   string
-	DNSZoneRG, DNSZoneSub, DNSZoneDomain, DNSRecordID string
-	DNSZonePrivate                                    bool
+	PrivateZoneIds, PublicZoneIds                     []string
```

Comment:
It doesn't make sense to have these be separate structures. Group similar information with a struct

struct PrivateZone {
ids []string
sub string
rg string
}

Reply:
done